src/Uno.UI/UI/Xaml/Controls/Frame/Frame.partial.mux.cs

```diff
@@ -48,6 +49,17 @@ protected override void OnApplyTemplate()
 
 		base.OnApplyTemplate();
 
+#if HAS_UNO
+		if (this.TemplatedRoot?.GetType().Name?.Contains("NativeFramePresenter", StringComparison.Ordinal) == true)
```

Comment:
Not sure if I'm following, but maybe
```suggestion
		if (useWinUIBehavior && this.TemplatedRoot?.GetType().Name?.Contains("NativeFramePresenter", StringComparison.Ordinal) == true)
```
?